repotracker/wrappers.go

```diff
@@ -68,10 +68,6 @@ func CollectRevisionsForProject(ctx context.Context, conf *evergreen.Settings, p
 }
 
 func ActivateBuildsForProject(project model.ProjectRef) error {
-	if !project.IsEnabled() {
```

Comment:
I think we still want this check so that disabled projects don't run old tasks wouldn't we?

Reply:
This check exists already in the caller of ActivateBuildsForProject, so it's redundant.

Reply:
Since it's a public function there could be more callers in the future though. I don't think it's a bad thing that it's checked twice since everything is in memory already